collect_app/src/main/java/org/odk/collect/android/preferences/screens/ServerPreferencesFragment.java

```diff
@@ -140,6 +140,12 @@ public void addAggregatePreferences() {
         serverUrlPreference.setOnPreferenceChangeListener(createChangeListener());
         serverUrlPreference.setSummary(serverUrlPreference.getText());
 
+        serverUrlPreference.setOnBindEditTextListener(editText -> {
+            editText.selectAll();
```

Comment:
Although it's only one line, it would be good for this new feature to have a test to validate that it works correctly and to make sure it doesn't get removed in the future. If you have experience writing tests in Android, then let me know here how you're thinking of testing this, but otherwise I can just add a test myself as part of this PR.

Reply:
@seadowg Thanks. Appreciate it if you could write a test for this.